ui/framework/src/ui-framework/ui-items-provider/StandardStatusbarItemsProvider.tsx

```diff
@@ -0,0 +1,98 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module UiItemsProvider
+ */
+
+import * as React from "react";
+import { CommonStatusBarItem, StageUsage, StatusBarSection, UiItemsManager, UiItemsProvider } from "@bentley/ui-abstract";
+import { StatusBarItemUtilities } from "../statusbar/StatusBarItemUtilities";
+import { ToolAssistanceField } from "../statusfields/toolassistance/ToolAssistanceField";
+import { withStatusFieldProps } from "../statusbar/withStatusFieldProps";
+import { MessageCenterField } from "../statusfields/MessageCenter";
+import { withMessageCenterFieldProps } from "../statusbar/withMessageCenterFieldProps";
+import { ActivityCenterField } from "../statusfields/ActivityCenter";
+import { SnapModeField } from "../statusfields/SnapMode";
+import { SelectionInfoField } from "../statusfields/SelectionInfo";
+import { TileLoadingIndicator } from "../statusfields/tileloading/TileLoadingIndicator";
+import { SelectionScopeField } from "../statusfields/SelectionScope";
+
+/** Defines what tools to include
+ * @beta
+ */
+export interface DefaultStatusbarItems {
+  toolAssistance?: boolean;
+  messageCenter?: boolean;
+  activityCenter?: boolean;
+  accuSnapModePicker?: boolean;
+  tileLoadIndicator?: boolean;
+  selectionScope?: boolean;
+  selectionInfo?: boolean;
+}
+
+/** Provide standard tools for the SimpleStatusbarWidget
+ * @beta
+ */
+export class StandardStatusbarItemsProvider implements UiItemsProvider {
+  public static providerId = "uifw:StandardStatusbarItemsProvider";
+  public readonly id = StandardStatusbarItemsProvider.providerId;
+
+  constructor(private _defaultItems?: DefaultStatusbarItems, private _isSupportedStage?: (stageId: string, stageUsage: string, stageAppData?: any) => boolean) { }
+
+  public static register(defaultItems?: DefaultStatusbarItems, isSupportedStage?: (stageId: string, stageUsage: string, stageAppData?: any) => boolean) {
+    UiItemsManager.register(new StandardStatusbarItemsProvider(defaultItems, isSupportedStage));
+  }
+
+  public static unregister() {
+    UiItemsManager.unregister(StandardStatusbarItemsProvider.providerId);
+  }
+
+  public provideStatusBarItems(stageId: string, stageUsage: string, stageAppData?: any): CommonStatusBarItem[] {

```

Comment:
Looks good, maybe we want to keep order similar to how we have it by default in the viewer? 
https://github1s.com/iTwin/viewer/blob/HEAD/packages/modules/viewer-react/src/components/app-ui/statusbars/AppStatusBar.tsx

Reply:
I am rearranging the first few items to match. Note the ClearEmphasis is no longer need now that the clear button is dynamically added to horizontal toolbar. The SnapMode picker is remaining in the center section so other editing specific status fields can sit in that same area.